docs/common/guides/jlink_image.adoc

```diff
@@ -0,0 +1,188 @@
+
+[source,bash,subs="attributes+"]
+----
+mvn archetype:generate -DinteractiveMode=false \
+    -DarchetypeGroupId=io.helidon.archetypes \
+    -DarchetypeArtifactId=helidon-quickstart-{helidon-lc-flavor} \
+    -DarchetypeVersion={helidon-version} \
+    -DgroupId=io.helidon.examples \
+    -DartifactId=helidon-quickstart-{helidon-lc-flavor} \
+    -Dpackage=io.helidon.examples.quickstart.{helidon-lc-flavor}
+----
+
+The archetype generates a Maven project in your current directory
+(for example, `helidon-quickstart-{helidon-lc-flavor}`). Change into this directory and build.
+
+[source,bash,subs="attributes+"]
+----
+cd helidon-quickstart-{helidon-lc-flavor}
+mvn package
+----
+
+At this point you can run the application using the JVM:
+
+[source,bash,subs="attributes+"]
+----
+java -jar target/helidon-quickstart-{helidon-lc-flavor}.jar
+----
+
+In another shell test an endpoint:
+
+[source,bash]
+----
+curl -X GET http://localhost:8080/greet
+----
+
+The application should respond with `{"message":"Hello World!"}`
+
+Now stop the running application (by pressing Ctrl+C).
+
+For more information about the Quickstart application and other enpoints it supports see the
+<<{helidon-lc-flavor}/guides/02_quickstart.adoc,Helidon {helidon-uc-flavor} quickstart Guide>>.
+
+== Building a Custom Runtime Image
+
+You can build a custom runtime image in 2 different ways:
+
+* Locally, on your desktop
+* Using Docker
+
+=== Local build
+
+Build the custom runtime image using the jlink image profile:
+
+[source,bash]
+----
+mvn package -Pjlink-image
+----
+
+[TIP]
+This uses the `helidon-maven-plugin` to perform the custom image generation.
+
+After the build completes it will report some statistics about the build including
+the reduction in image size.
+
+The `target/helidon-quickstart-{helidon-lc-flavor}` directory is a self contained
+custom image of your application. It contains your application, its runtime
+dependencies and the JDK modules it depends on. You can start your application
+using the provide
+`start` script:
+
+[source,bash,subs="attributes+"]
+----
+./target/helidon-quickstart-{helidon-lc-flavor}/bin/start
+----
+
+=== Class Data Sharing (CDS) Archive
+
+Aslo included in the custom image is a Class Data Sharing (CDS) archive that
+improves your application's startup performance and in-memory footprint.
+You can learn more about Class Data Sharing in the
+https://docs.oracle.com/en/java/javase/11/vm/class-data-sharing.html[JDK documentation].
+
+The CDS archive increases your image size to get these performance optimizations.
+It can be of significant size (tens of MB). The size of the CDS archive is
+reported at the end of the build output.
+
+If you'd rather have a smaller image size (with a slightly increased startup time) you
+can skip the creation of the CDS archive by executing your build like this:
+
+[source,bash]
+----
+mvn package -Pjlink-image -Djlink.image.addClassDataSharingArchive=false
+----
+
+For more information see the
```

Comment:
Since those docs also say to add a profile, which would be confusing here since it is already present, maybe say "For more information on available configuration options see the"

Reply:
fixed